libraries/core_libs/consensus/src/vote_manager/vote_manager.cpp

```diff
@@ -33,6 +33,14 @@ void VoteManager::retreieveVotes_() {
   LOG(log_si_) << "Retrieve verified votes from DB";
   auto verified_votes = db_->getVerifiedVotes();
   const auto pbft_step = db_->getPbftMgrField(PbftMgrRoundStep::PbftStep);
+  for (auto const& v : verified_votes) {
+    addVerifiedVote(v);
+    LOG(log_dg_) << "Retrieved verified vote " << *v;
+  }
+
+  // TODO: Implement votes sync properly
+  // Since this is invoked on startup no peers are connected yet, invoke two minutes after startup
+  thisThreadSleepForSeconds(120);
   for (auto const& v : verified_votes) {
     // Rebroadcast our own next votes in case we were partitioned...
     if (v->getStep() >= FIRST_FINISH_STEP && pbft_step > EXTENDED_PARTITION_STEPS) {
```

Comment:
hmm, and what about `pbft_step > EXTENDED_PARTITION_STEPS`. With back-off this definitely does not make sense, as we were not reaching steps above 1k

Reply:
all of our consensus nodes are now over 1000, they are currently on step 8000 but yes with the backoff 1000 is too much